db/migrate/20210106133648_users_post_foreign_key.rb

```diff
@@ -0,0 +1,5 @@
+class UsersPostForeignKey < ActiveRecord::Migration[6.0]
+  def change
+    add_column :post, :user_id, foreign_key:true
+  end
+end
```

Comment:
- [ ] The error seems to be coming from this migration file. It is difficult to know what you are trying to achieve in this migration from the name you gave it, but I suspect the issue is most likely because the migration is not set correctly. Please look into it and update accordingly.